src/applications/gi/containers/LandingPage.jsx

```diff
@@ -136,15 +133,9 @@ export function LandingPage({
     setSearchError(searchQuery === '');
   };
 
-  const buttonLabel = gibctSearchEnhancements ? 'Search' : 'Search Schools';
-
-  let searchLabel = gibctSearchEnhancements
+  const searchLabel = environment.isProduction()
```

Comment:
Can we add the prod flag comment to identify the matching story?

Reply:
Yes, I didn't mean to lose that.